src/components/Forecast/Widget.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import Form from './Form/Form'
+import WeatherDetailsToday from './WeatherDetailsToday/WeatherDetailsToday'
+import FiveDayForecast from './FiveDayForecast/FiveDayForecast'
+
+import './Widget.scss'
+
+const myAPIKey = 'dce6d076b7a2ffe64e08efe5ff779fa3'
+
+class Widget extends React.Component {
+  constructor(props) {
+    super(props)
+    this.getWeatherData = this.getWeatherData.bind(this)
+    this.goback = this.goback.bind(this)
```

Comment:
These two can be removed by using arrow functions, that will automatically use the lexical `this`

So below, when you defined the function, instead of writing:
```javascript
getWeatherData(event) {}
```
You write
```javascript
getWeatherData = (event) => {}
```
And then adding `this.getWeatherData = this.getWeatherData.bind(this)` to the constructor is no longer necessary!